### PR TITLE
Updated README example with v5+ information

### PR DESCRIPTION
**Pull request by JuanMaRuiz**
Example in the README is not correct after the migration to v5. 

I have updated it using the proper values.

Hope it helps.

Best regards.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,21 +23,20 @@ const psi = require('psi');
 
 (async () => {
   // Get the PageSpeed Insights report
-  const data = await psi('https://theverge.com');
-  console.log(data.ruleGroups.SPEED.score);
-  console.log(data.pageStats);
+  const { data } = await psi('https://theverge.com');
+  console.log('Speed score:', data.lighthouseResult.categories.performance.score);
+  // console.log(data.pageStats);
```
Comment (exterkamp) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -23,21 +23,20 @@ const psi = require('psi');
...
 
   // Output a formatted report to the terminal
   await psi.output('https://theverge.com');
   console.log('Done');
 
-  // Supply options to PSI and get back speed and usability scores
-  const data2 = await psi('https://theverge.com', {
+  // Supply options to PSI and get back speed
+  const data2 = await psi('https://google.com', {
```
Comment (exterkamp) [inline README.md]: Either keep both theverge, or change both to google. I have no preference.

File: README.md (hunk 1 above)
Comment (JuanMaRuiz) [reply README.md]: Sorry about it. I forgot to delete 🤦it.

File: README.md (hunk 2 above)
Comment (JuanMaRuiz) [reply README.md]: Oks

**Comment by JuanMaRuiz** [comment]
> LGTM! A few nits, but great job updating this!

**Comment by JuanMaRuiz** [comment]
Hi @exterkamp all requested changes are done. Hope everything is ok now.

Best regards

**Comment by exterkamp** [comment]
Build is failing with `FAILED_DOCUMENT_REQUEST` & `(Details: net::ERR_ACCESS_DENIED)` which is a flake afaict.

**Pull request merged**